server2/routes/confirmationRouter.js

```diff
@@ -0,0 +1,51 @@
+var express = require("express");
+var router = express.Router();
+var User = require("../models/user");
+var EmailToken = require("../models/emailToken");
+
+router
+    .route("/:token")
+    .get((req, res, next) => {
+        try {
+            const getToken = () => {
+                return new Promise((resolve, reject) => {
+                    resolve(EmailToken.findOne());
+                })
+            };
+
+            const verify = async () => {
+                const tokenObj = await getToken();
+                const token = tokenObj.token;
+
+                const jsonReceived = decodeURIComponent(req.params.token);
+                const unpacked = JSON.parse(jsonReceived);
+
+                const recToken = unpacked.token;
+
+                if (recToken == token) {        
+                    console.log("Token confirmed.");
+                    User.create(unpacked.user);
+                    
+                    // Confirm successful page
+                    res.send("Your Email is confirmed!");
```

Comment:
that is it? Shouldn't we display a success page or even automatically redirect the user back to the main page?